docs/14.Supported_vanilla_objects.md

```diff
@@ -0,0 +1,172 @@
+## Supported vanilla objects
+
+### Message Objects
+```
+!=
+%
+&
+&&
+*
++
+-
+/
+<
+<<
+<=
+==
+>
+>=
+>>
+abs
+atan
+atan2
+b
+bang
+bendin
+bendout
+bng
+change
+clip
+cnv
+cos
+ctlin
+ctlout
+dbtopow
+dbtorms
+declare
+del
+delay
+div
+exp
+f
+float
+floatatom
+ftom
+hradio
+hsl
+i
+inlet
+int
+line
+loadbang
+log
+makenote
+max
+metro
+min
+mod
+moses
+mtof
+nbx
+notein
+noteout
+outlet
+pack
+pgmin
+pgmout
+pipe
+poly
+pow
+powtodb
+print
+r
+random
+receive
+rmstodb
+route
+s
+sel
+select
+send
+sin
+spigot
+sqrt
+swap
+symbol
+symbolatom
+t
+table
+tabread
+tabwrite
+tan
+tgl
+timer
+touchin
+touchout
+trigger
+unpack
+until
+vradio
+vsl
+wrap
+```
+
+### Signal Objects
+```
+*~
++~
+-~
+/~
+abs~
+adc~
+biquad~
+bp~
+catch~
+clip~
+cos~
+cpole~
+czero_rev~
+czero~
+dac~
+dbtopow~
+dbtorms~
+delread~
+delwrite~
+env~
+exp~
+ftom~
+hip~
+inlet~
+line~
+lop~
+max~
+min~
+mtof~
+noise~
+osc~
+outlet~
+phasor~
+powtodb~
+pow~
+q8_rsqrt~
+q8_sqrt~
+receive~
+rmstodb~
+rpole~
+rsqrt~
+rzero_rev~
+rzero~
+r~
+samphold~
+samplerate~
+send~
+sig~
+snapshot~
+sqrt~
+s~
+tabosc4~
+tabread4~
+tabread~
+tabwrite~
+throw~
+vcf~
+vd~
+wrap~
+```
+#### `-p` Search Paths
```

Comment:
This is a great list to add, thank you. I'd remove this section though. Don't use a `-p` flag, use a `[declare -path]` object in your patch instead. The `-p` flag isn't implemented properly and we never used it.

Reply:
Right. I was just assuming that this solution works , but didn't have the time to test it yet 
https://github.com/giuliomoro/hvcc/commit/50329d8ed4b35573be02fdf563c99c4815d5bf22